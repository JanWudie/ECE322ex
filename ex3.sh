#!/bin/bash

# Filename: ex3.sh
#
# Problem: Make a script to check if a file exists. The filename is the first 
# argument passed to the script. Print that the file exists if it does, and 
# file doesn't exist if it doesn't.

#this should work, did in my own testing, not sure what's up with your check!
VAR=test/ex3.arg
if [ -a $VAR ]; then
 echo The file \"$VAR\" exists.
else
 echo The file \"$VAR\" does not exist.
fi
