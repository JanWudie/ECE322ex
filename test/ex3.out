The file "test/ex3.arg" exists.
The file "test/ex3.arg" exists.
